### PR TITLE
Create themes.ready method

### PR DESCRIPTION
**Pull request by anna-zhernovkova**

### CODE REVIEW

**Comment by AlekseyMartynov** [comment]
Can we possibly keep `loadCallback`?
Though undocumented, we have recommended it sometimes.
Example: https://devexpress.com/issue=T499870

**Comment by anna-zhernovkova** [comment]
It's kept:
` 
if(loadCallback) {
    themeReadyCallback.add(loadCallback);
}`
(themes.js, line 203)

**Comment by AlekseyMartynov** [comment]
Ok, cool

**Comment by AlekseyMartynov** [comment]
Lifting `timerId` is a potential bug. What if exported `waitForThemeLoad` is called twice? Don't we lose a timer?

File: js/ui/themes.js [hunk 1]
```diff
@@ -292,11 +300,13 @@ function detachCssClasses(element) {
...
 init({
     _autoInit: true,
-    loadCallback: function() {
-        themeReadyCallback.fire();
-    }
+    _forceTimeout: true
 });
 
+function themeReady(callback) {
+    themeReadyCallback.add(callback);
```
Comment (ovchinnikov) [inline js/ui/themes.js:307]: Callback will never call if theme was loaded early. Is it expected behaviour?

**Comment by anna-zhernovkova** [comment]
`waitForThemeLoad` - it is a private method. It doesn't use in our code and WidgetsGallery, so, I think, that it needs only for tests

File: js/ui/themes.js (hunk 1 above)
Comment (anna-zhernovkova) [reply js/ui/themes.js:307]: Yes. We need to subscribe on `themes.ready` before themes loading

**Review by ovchinnikov — APPROVED** [review]

**Review by AlekseyMartynov — COMMENTED** [review]
LGTM

**Pull request merged**